test/check-machines-create

```diff
@@ -593,23 +593,28 @@ vnc_password= "{vnc_passwd}"
                                                       storage_size=128, storage_size_unit='MiB',
                                                       start_vm=False))
 
-        # Test detection of ISO file in URL
-        runner.createTest(TestMachinesCreate.VmDialog(self, sourceType='url',
-                                                      location=config.ISO_URL,
-                                                      memory_size=128, memory_size_unit='MiB',
-                                                      storage_pool="NoStorage",
-                                                      start_vm=True))
+        self.browser.eval_js('window.debugging = "machines"')
```

Comment:
Whoops, debugging leftover. Although it's quite nice to investigate the arch failure :grin: